src/Database/Schema/Table.php

```diff
@@ -14,789 +14,9 @@
  */
 namespace Cake\Database\Schema;
 
-use Cake\Database\Exception;
-use Cake\Database\Type;
-use Cake\Datasource\ConnectionInterface;
-
 /**
- * Represents a single table in a database schema.
- *
- * Can either be populated using the reflection API's
- * or by incrementally building an instance using
- * methods.
- *
- * Once created Table instances can be added to
- * Schema\Collection objects. They can also be converted into SQL using the
- * createSql(), dropSql() and truncateSql() methods.
+ * @deprecated Provided for backward compatibility. Use TableSchema instead.
  */
-class Table
+class Table extends TableSchema
```

Comment:
@thinkingmedia Indeed you did change `ORM\Cake\Table`. Then does the line 17 in your capture really works? It seems to change the signature of parent method.
```php
// parent signature
protected function _initializeSchema(\Cake\Database\Schema\TableSchema $schema);
// child signature
protected function _initializeSchema(\Cake\Database\Schema\Table $schema);
```
Even if it works, the following code wouldn't work:
```php
use Cake\Database\Schema\Table as Schema;

$schema = $articles->schema(); // \Cake\Database\Schema\TableSchema
if (!$schema instanceof Schema) { // \Cake\Database\Schema\Table
    throw new \Exception('Never come here');
} 
```